org.viking.Viking.appdata.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
FYI, please consider maintaining this code in the upstream repo – it's not specific to Flathub and other distributions/stores could benefit from it too.

Reply:
I will be including all related code in the upstream repo, once the refinements here are completed and suitable for publication.